From 6386303273abb05226785ceae19ba7a70255c2f5 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 12 Dec 2021 01:02:38 +0000
Subject: [PATCH] gnu: hcl: Update to 1.0.0

* gnu/packages/golang:
  Use the latest tag for HCL V1 instead of commit
  [native-inputs]: add go-github-com-davecgh-go-spew
---
 gnu/packages/golang.scm | 46 ++++++++++++++++++++---------------------
 1 file changed, 23 insertions(+), 23 deletions(-)

diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 699bc49401..d3cbf14ff7 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -2334,31 +2334,31 @@ (define-public go-github-com-tj-docopt
     (license license:expat)))
 
 (define-public go-github-com-hashicorp-hcl
-  (let ((commit "23c074d0eceb2b8a5bfdbb271ab780cde70f05a8")
-        (revision "0"))
-    (package
-      (name "go-github-com-hashicorp-hcl")
-      (version (git-version "0.0.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                       (url "https://github.com/hashicorp/hcl")
-                       (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                  (base32
-                    "0db4lpqb5m130rmfy3s3gjjf4dxllypmyrzxv6ggqhkmwmc7w4mc"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:tests? #f
-         #:import-path "github.com/hashicorp/hcl"))
-      (synopsis "Go implementation of HashiCorp Configuration Language")
-      (description
-       "This package contains the main implementation of the @acronym{HCL,
+  (package
+    (name "go-github-com-hashicorp-hcl")
+    (version "1.0.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/hashicorp/hcl")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32
+         "0q6ml0qqs0yil76mpn4mdx4lp94id8vbv575qm60jzl1ijcl5i66"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/hashicorp/hcl"))
+    (native-inputs
+     `(("go-github-com-davecgh-go-spew" ,go-github-com-davecgh-go-spew)))
+    (synopsis "Go implementation of HashiCorp Configuration Language V1")
+    (description
+     "This package contains the main implementation of the @acronym{HCL,
 HashiCorp Configuration Language}.  HCL is designed to be a language for
 expressing configuration which is easy for both humans and machines to read.")
-      (home-page "https://github.com/hashicorp/hcl")
-      (license license:mpl2.0))))
+    (home-page "https://github.com/hashicorp/hcl")
+    (license license:mpl2.0)))
 
 (define-public go-github-com-hashicorp-hcl-v2
   (package
-- 
2.34.0

